solr/seek/conf/mapping-hyphen.txt

```diff
@@ -0,0 +1,7 @@
+# Mapping of Unicode hyphen to ASCII hyphen
+"\u2010" => "-"
+"\u2011" => "-"
+"\u2012" => "-"
+"\u2013" => "-"
+"\u2014" => "-"
+"\u2015" => "-"
```

Comment:
I don't this file is needed, as they are already included in teh FoldToASCII

Reply:
i just realized that